docs/administering/for-work.md

```diff
@@ -0,0 +1,112 @@
+other systems.
+
+SAML is a protocol for exchanging information about access control. SAML support is coming soon. If
+you need this feature, please feel free to [request a feature key
+today][https://sandstorm.io/business]. SAML support should be compatible with Shibboleth and other
+systems.
+
+To enable either of these login providers, take the following steps:
+
+- Log into your Sandstorm server.
+
+- Make sure you have enabled [Sandstorm for Work on your server](#enabling-sandstorm-for-work).
+
+- Click **Admin Settings** within your Sandstorm server; this should take you to `/admin/settings`.
+
+You should now see a checkbox that allows you to enable LDAP login.
+
+### Features coming soon
+
+We're still working on the following features:
+
+**Organization management.** Choose an LDAP group that, upon login, should automatically be given
+_user_ status on this Sandstorm server.
+
+**Group Management.** This will allow you to share a grain with everyone in an LDAP/Active Directory
+group, such as the marketing department.
+
+**Global Access Control.** Configure organization-wide access control policies, such as prohibiting
+your employees from sharing grains outside of the organization.
+
+**Global Audit Logging.** Monitor data across the whole organization to keep track of who has
+accessed what.
+
+If you're interested in these features, we'd love to hear from you. Please [contact
+us](https://sandstorm.io/business)!
+
+## How to get Sandstorm for Work
+
+### Installing Sandstorm for Work
+
+Sandstorm comes bundled with the code needed to convert it into a Sandstorm for Work installation.
+
+If you don't have Sandstorm yet, follow these steps first.
+
+- [Install Sandstorm](https://sandstorm.io/install) via the usual directions.
+
+- Obtain a feature key. At the moment, we send all feature keys by email to you. To request a key,
+  visit the [Sandstorm for Work page on our website](https://sandstorm.io/business), click **Contact
+  Us**, fill out the form, and mention that you want to unlock Sandstorm for Work.
+
+Now that you have Sandstorm, enable Sandstorm for Work. These steps work properly on any up-to-date
+Sandstorm install, even if you installed before Sandstorm for Work was avaiable.
+
+- Log in as an administrator.
+
+- Click on your name, then click on **Admin Settings** to visit `/admin/settings`.
+
+- Click on **For Work** to visit `/admin/features`.
+
+- Click **Choose File** to upload a feature key file. You can alternatively paste a feature key into
+  the text area and click Verify.
+
+Once you've done this, you will see information about your feature key on the **For Work** tab. All
+Sandstorm features enabled by your feature key will be available to you in the most
+contextually-relevant location.
+
+### All about feature keys
+
+A feature key is a text file of this format:
+
+```
+--------------------- BEGIN SANDSTORM FEATURE KEY ----------------------
```

Comment:
For those who are curious: This is a truncated version of my ~/.ssh/id_rsa.pub . :)